packages/chat/package.json

```diff
@@ -0,0 +1,119 @@
+{
+  "name": "@voiceflow/chat",
```

Comment:
🟢 I would add `"private": true` for now so we don't start publishing it immediately just in case we decide to change the name or something

Reply:
sure, will do, thanks